pom.xml

```diff
@@ -337,14 +343,14 @@
             <groupId>org.lz4</groupId>
             <artifactId>lz4-java</artifactId>
             <!-- should match version from drivers POM -->
-            <version>1.7.1</version>
+            <version>1.8.0</version>
         </dependency>
 
         <dependency>
             <groupId>org.xerial.snappy</groupId>
             <artifactId>snappy-java</artifactId>
             <!-- should match version from drivers POM -->
-            <version>1.1.10.5</version>
+            <version>1.1.10.8</version>
```

Comment:
Does not match, but it probably still works.

Reply:
Does not match to what ?